chemprop/args.py

```diff
@@ -84,6 +84,11 @@ class CommonArgs(Tap):
     """Number of workers for the parallel data loading (0 means sequential)."""
     batch_size: int = 50
     """Batch size."""
+    atom_descriptors: Literal['None', 'feature', 'descriptor'] = 'None'
```

Comment:
Is there a reason that you use `'None'` instead of `None`? I think this works fine but I might do the following instead since `None` is a better indicator that something is not being used than the string `'None'`.

```suggestion
    atom_descriptors: Literal['feature', 'descriptor'] = None
```